test-http/src/test/cve_id.py

```diff
@@ -156,6 +165,160 @@ def test_post_cve_id_empty_short_name():
+    """ cve services rejects year that isn't a 4 digit number"""
+    res = requests.post(
+        f'{env.AWG_BASE_URL}{CVE_ID_URL}',
+        headers=utils.BASE_HEADERS,
+        params={
+            'amount': '10',
+            'batch_type': 'sequential',
+            'cve_year': '20111',
+            'short_name': 'mitre'
+        }
+    )
+    assert res.status_code == 400
+    response_contains_json(res, 'error', 'BAD_INPUT')
+
+
+# general user test
+def test_post_cve_id_empty_amount():
+    """ cve services rejects empty amount """
+    res = requests.post(
+        f'{env.AWG_BASE_URL}{CVE_ID_URL}',
+        headers=utils.BASE_HEADERS,
+        params={
+            'amount': '',
+            'batch_type': 'sequential',
+            'cve_year': f'{utils.CURRENT_YEAR}',
+            'short_name': 'mitre'
+        }
+    )
+    assert res.status_code == 400
+    response_contains_json(res, 'error', 'BAD_INPUT') 
+
+
+# general user test
+def test_post_cve_id_invalid_amount():
+    """ cve services rejects amount less than or equal to 0 """
+    res = requests.post(
+        f'{env.AWG_BASE_URL}{CVE_ID_URL}',
+        headers=utils.BASE_HEADERS,
+        params={
+            'amount': '-1',
+            'batch_type': 'sequential',
+            'cve_year': f'{utils.CURRENT_YEAR}',
+            'short_name': 'mitre'
+        }
+    )
+    assert res.status_code == 400
+    response_contains_json(res, 'error', 'INVALID_AMOUNT')
+
+
+# general user test
+def test_post_cve_id_no_batch_type():
+    """ cve services rejects not having a batch type"""
+    res = requests.post(
+        f'{env.AWG_BASE_URL}{CVE_ID_URL}',
+        headers=utils.BASE_HEADERS,
+        params={
+            'amount': '10',
+            'batch_type': '',
+            'cve_year': f'{utils.CURRENT_YEAR}',
+            'short_name': 'mitre'
+        }
+    )
+    assert res.status_code == 400
+    response_contains_json(res, 'error', 'NO_BATCH_TYPE')
+
+
+# general user test
+def test_post_cve_id_invalid_batch_type():
+    """ cve services rejects having an invalid batch type"""
+    res = requests.post(
+        f'{env.AWG_BASE_URL}{CVE_ID_URL}',
+        headers=utils.BASE_HEADERS,
+        params={
+            'amount': '10',
+            'batch_type': '---',
+            'cve_year': f'{utils.CURRENT_YEAR}',
+            'short_name': 'mitre'
+        }
+    )
+    assert res.status_code == 400
+    response_contains_json(res, 'error', 'INVALID_BATCH_TYPE')
+
+
+# general user test
+def test_post_cve_id_cna_user():
+    """ cna and user must exist """
+    res = requests.post(
+        f'{env.AWG_BASE_URL}{CVE_ID_URL}',
+        headers=utils.BASE_HEADERS,
+        params={
+            'amount': '10',
+            'batch_type': 'sequential',
+            'cve_year': f'{utils.CURRENT_YEAR}',
+            'short_name': 'mitre'
+        }
+    )
+    assert res.status_code == 200
+    assert 'cna' in res.content.decode()
+    assert 'user' in res.content.decode()
```

Comment:
This looks like its just testing sequential reservation. There already exists a test `test_post_cve_id_reservation` that tests both sequential and non-sequential. If each of the responses in that test needs to contain the same `cna` and `user` content, I would suggest updating that test instead of adding this separate test that only looks at one sequential reservation.

Reply:
If you can, double check that all of these new tests aren't testing the same thing that any of the existing tests are testing.